### PR TITLE
Update Wiz.cn.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/Wiz.cn.xml [hunk 1]
```diff
@@ -26,12 +23,14 @@
...
 	<target host="blog.wiz.cn" />
 	<target host="cdn.wiz.cn" />
-	<target host="get.wiz.cn" />
-	<!-- Find get.wiz.cn tests at: https://www.wiz.cn/downloads-windows.html -->
-	<target host="s2.wiz.cn" />
+	<target host="cdn2.wiz.cn" />
+	<target host="get.wiz.cn" /><!-- Find get.wiz.cn tests at: https://www.wiz.cn/downloads-windows.html -->
 	<target host="url.wiz.cn" />
 
+	<rule from="^http://wiz\.cn/" to="https://www.wiz.cn/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Wiz.cn.xml]: Why is this required?

Comment (ivysrono) [reply src/chrome/content/rules/Wiz.cn.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/418257579#L599

Comment (J0WI) [reply src/chrome/content/rules/Wiz.cn.xml]: Please add a hint in the comments.
